e2e/node/e2e.test.ts

```diff
@@ -148,37 +148,92 @@ describe("End-to-end verifiable credentials tests for environment", () => {
 
   describe("lookup VCs", () => {
     it("returns all VC issued matching a given shape", async () => {
-      const result = await getVerifiableCredentialAllFromShape(
-        new URL("derive", env.vcProvider).href,
-        {
-          "@context": [
-            "https://www.w3.org/2018/credentials/v1",
-            "https://schema.inrupt.com/credentials/v1.jsonld",
-          ],
-          type: ["VerifiableCredential", "SolidAccessGrant"],
-          credentialSubject: {
-            id: vcSubject,
-            providedConsent: {
-              hasStatus:
-                "https://w3id.org/GConsent#ConsentStatusExplicitlyGiven",
+      const [credential1, credential2] = await Promise.all([
+        issueVerifiableCredential(
+          new URL("issue", env.vcProvider).href,
+          validSubjectClaims({ resource: "https://example.org/some-resource" }),
+          validCredentialClaims,
+          {
+            fetch: session.fetch,
+          },
+        ),
+        issueVerifiableCredential(
+          new URL("issue", env.vcProvider).href,
+          validSubjectClaims({
+            resource: "https://example.org/another-resource",
+          }),
+          validCredentialClaims,
+          {
+            fetch: session.fetch,
+          },
+        ),
+      ]);
+      await expect(
+        getVerifiableCredentialAllFromShape(
+          new URL("derive", env.vcProvider).href,
+          {
+            "@context": [
+              "https://www.w3.org/2018/credentials/v1",
+              "https://schema.inrupt.com/credentials/v1.jsonld",
+            ],
+            type: ["VerifiableCredential", "SolidAccessGrant"],
+            credentialSubject: {
+              id: vcSubject,
+              providedConsent: {
+                hasStatus:
+                  "https://w3id.org/GConsent#ConsentStatusExplicitlyGiven",
+              },
             },
           },
+          {
+            fetch: session.fetch,
+          },
+        ),
+      ).resolves.not.toHaveLength(0);
+
+      await expect(
+        getVerifiableCredentialAllFromShape(
+          new URL("derive", env.vcProvider).href,
+          credential1,
+          {
+            fetch: session.fetch,
+          },
+        ),
+      ).resolves.toHaveLength(1);
+
+      await expect(
+        getVerifiableCredentialAllFromShape(
+          new URL("derive", env.vcProvider).href,
+          credential2,
+          {
+            fetch: session.fetch,
+          },
+        ),
+      ).resolves.toHaveLength(1);
+
+      await revokeVerifiableCredential(
+        new URL("status", env.vcProvider).href,
+        credential1.id,
+        {
+          fetch: session.fetch,
         },
+      );
+
+      await revokeVerifiableCredential(
+        new URL("status", env.vcProvider).href,
+        credential2.id,
         {
           fetch: session.fetch,
         },
       );
```

Comment:
This can be done in parallel / a `Promise.all`

Reply:
Good point, fixed in https://github.com/inrupt/solid-client-vc-js/pull/870/commits/aa9973a0d77ca355a835e2b3729d43cf43354510. I hesitated to put the asyn assertions in a similar wrapper, but I decided not to because it could make it harder to figure out which assertion failed.